### PR TITLE
Add readme and build scripts from other repo for initial setup and testing

### PR DESCRIPTION
**Pull request by michael-hawker**

### CODE REVIEW

File: Contributing.md
Comment (Arlodotexe) [inline Contributing.md:1]: Do we want to update this for the tooling repo? It's still branded as Labs

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+## ❔ Questions <a name="question"></a>
+
+Due to the high volume of incoming issues please keep our GitHub issues for bug reports about the Labs infrastructure itself. For general questions and discussions (including new features), [please open up a discussion (if one doesn't exist already)](https://github.com/CommunityToolkit/Labs-Windows/discussions). You can [also reach out in the `#community-toolkit` channel on Discord here](https://aka.ms/wct/discord)
+
+## 🙋 Help Wanted <a name="help"></a>
+
+We have a list of issues that are labeled as [help wanted](https://github.com/CommunityToolkit/Labs-Windows/labels/help%20wanted). Note, the level of complexity in the list can vary, but feel free to jump in to help solve these issues. Be sure to comment on issues first where you'd like to help out to coordinate with others in the community.
```
Comment (Arlodotexe) [inline Contributing.md:15]:
```suggestion
We have a list of issues that are labeled as [help wanted](https://github.com/CommunityToolkit/Tooling-Submodule/labels/help%20wanted). Note, the level of complexity in the list can vary, but feel free to jump in to help solve these issues. Be sure to comment on issues first where you'd like to help out to coordinate with others in the community.
```

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Please bear with us as we onboard to this new process using Windows Community Toolkit Labs.
+
+**Note:** _In Preview we're currently not accepting new experiments beyond our trial list to ensure our infrastructure is stabilized, if you'd like to contribute [please see where you can lend a hand with Labs itself here](https://github.com/CommunityToolkit/Labs-Windows/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)_ [If you have an idea for a component though, please feel free to open a discussion here.](https://github.com/CommunityToolkit/Labs-Windows/discussions?discussions_q=category%3AExperiments+category%3A%22Ideas%22+)
+
+## ❔ Questions <a name="question"></a>
+
+Due to the high volume of incoming issues please keep our GitHub issues for bug reports about the Labs infrastructure itself. For general questions and discussions (including new features), [please open up a discussion (if one doesn't exist already)](https://github.com/CommunityToolkit/Labs-Windows/discussions). You can [also reach out in the `#community-toolkit` channel on Discord here](https://aka.ms/wct/discord)
```
[the remark this answers is not in the document]
Comment (michael-hawker) [reply Contributing.md:11]: I tweaked the wording here, as I don't see us having discussions enabled on this repo?

File: Contributing.md (hunk 1 above)
Comment (michael-hawker) [reply Contributing.md:15]: Changing with the rest of changes locally

**Review by Arlodotexe — APPROVED** [review]

**Pull request merged**